valohai/distributed.py

```diff
@@ -0,0 +1,68 @@
+import json
+from typing import List, Optional
+
+from valohai import paths
+from valohai.internals.distributed_config import DistributedConfig, Member
+from valohai.internals.distributed_config.consts import MASTER_MEMBER_ID
+
+
+class Distributed:
+    """
+    Distributed toolkit accessed through `valohai.distributed`.
+    """
+
+    _config: Optional[DistributedConfig] = None
+
+    def is_distributed_task(self) -> bool:
+        # not a property to mimic `is_running_in_valohai`
+        try:
+            return bool(self.config.group_name)
+        except FileNotFoundError:
+            return False
+        except json.decoder.JSONDecodeError:
+            return False
```

Comment:
`JSONDecodeError` is re-exported from `json`, you don't generally need to look at the submodules.

```suggestion
        except (FileNotFoundError, json.JSONDecodeError):
            return False
```